doc/installing.rst

```diff
@@ -192,8 +191,7 @@ C code
 Significant speed gains in ``PypeIt`` can be enabled via compilation
 of the C code version of the b-spline fitting code. Compilation of
 the C code should happen automatically when you execute ``pip
-install`` or ``pip install -e .``. When installing from ``conda``,
-the C compilation has already been done for you. You can check that the C
+install`` or ``python setup.py develop``. You can check that the C
```

Comment:
Does `pip install -e .` not compile the C code?

Reply:
it should. if not, i would expect the tests to catch that. not sure why this got changed back to the old wording...